core/app/components/pubs/RemovePubForm.tsx

```diff
@@ -15,7 +15,7 @@ const PubRemoveForm = dynamic(
 			default: mod.PubRemoveForm,
 		}));
 	},
-	{ ssr: false, loading: () => <SkeletonCard /> }
+	{ loading: () => <SkeletonCard /> }
```

Comment:
i'm not so sure about my decision to just remove `ssr: false` here, i wonder if this still accomplishes the correct codesplitting behavior. if so, then this is fine